libs/helm/autocomplete/src/lib/hlm-autocomplete.ts

```diff
@@ -0,0 +1,202 @@
+import { BooleanInput } from '@angular/cdk/coercion';
+import {
+	booleanAttribute,
+	ChangeDetectionStrategy,
+	Component,
+	computed,
+	ElementRef,
+	forwardRef,
+	inject,
+	input,
+	linkedSignal,
+	output,
+	signal,
+} from '@angular/core';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { NgIcon, provideIcons } from '@ng-icons/core';
+import { lucideChevronDown, lucideSearch } from '@ng-icons/lucide';
+import { BrnCommandEmpty } from '@spartan-ng/brain/command';
+import { BrnDialogState } from '@spartan-ng/brain/dialog';
+import { ChangeFn, TouchFn } from '@spartan-ng/brain/forms';
+import { BrnPopover, BrnPopoverContent } from '@spartan-ng/brain/popover';
+import {
+	HlmCommand,
+	HlmCommandEmpty,
+	HlmCommandGroup,
+	HlmCommandItem,
+	HlmCommandList,
+	HlmCommandSearch,
+	HlmCommandSearchInput,
+} from '@spartan-ng/helm/command';
+import { HlmIcon } from '@spartan-ng/helm/icon';
+import { HlmPopoverContent } from '@spartan-ng/helm/popover';
+import { HlmAutocompleteTrigger } from './hlm-autocomplete-trigger';
+
+export const HLM_AUTOCOMPLETE_VALUE_ACCESSOR = {
+	provide: NG_VALUE_ACCESSOR,
+	useExisting: forwardRef(() => HlmAutocomplete),
+	multi: true,
+};
+
+let nextId = 0;
+
+@Component({
+	selector: 'hlm-autocomplete',
+	imports: [
+		BrnPopover,
+		BrnPopoverContent,
+		HlmPopoverContent,
+
+		HlmCommand,
+		HlmCommandSearch,
+		HlmCommandSearchInput,
+		HlmCommandList,
+		HlmCommandGroup,
+		HlmCommandItem,
+		BrnCommandEmpty,
+		HlmCommandEmpty,
+
+		HlmAutocompleteTrigger,
+
+		NgIcon,
+		HlmIcon,
+	],
+	providers: [HLM_AUTOCOMPLETE_VALUE_ACCESSOR, provideIcons({ lucideSearch, lucideChevronDown })],
+	template: `
+		<brn-popover
+			[align]="'start'"
+			[autoFocus]="'first-heading'"
```

Comment:
We can remove thing bindings for these static values.

```suggestion
			align="start"
			autoFocus="first-heading"
```